src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery.Test/AzureRm.Resources.ps1

```diff
@@ -0,0 +1,142 @@
+﻿function Get-AzureRmResourceGroup
```

Comment:
Why do you need to create your own AzureRM.Resources.ps1? Also, since the filename is 'Azure**Rm**.Resources.ps1' and not 'Azure**RM**.Resources.ps1', this will cause issues in NetStandard as Linux files are case-sensitive.

Reply:
AzureRM.Resources.ps1 is for doing operation which is not part ASR , but we need to do for infrastructure creation for Azure to Azure replication.
If I am not creating it what is the other way out ?  
'AzureRm.Resources.ps1' and not 'AzureRM.Resources.ps1' fixing this.